model/sparql/GenericSparql.php

```diff
@@ -542,34 +542,38 @@ public function queryConceptInfo($uris, $arrayClass = null, $vocabs = array(), $
      * @param string $lang
      * @return string sparql query
      */
-    private function generateQueryTypesQuery($lang) {
+    private function generateQueryTypesQuery(string $lang): string {
         $fcl = $this->generateFromClause();
+        $fromClause = $this->generateFromClause();
```

Comment:
Went with `FROM NAMED` since we already had a function for creating that clause :tada: